jenkins/node-image/Makefile

```diff
@@ -15,14 +15,39 @@
 IMG = gcr.io/k8s-testimages/kubekins-node
 TAG = $(shell date +v%Y%m%d)-$(shell git describe --tags --always --dirty)
 
+# default to 1.6 image
+VERSION:=1.6
+KUBEKINS_TAG:=1.6-v20161205-ad918bc
```

Comment:
IMO a :latest and :stable image or :latest-1.5 :stable-1.5 makes sense to me and is probably less of a nightmare